src/Meals.js

```diff
@@ -73,6 +95,28 @@ const mealPopUp = async (meal) => {
   `;
 
   mealDetailsContent.parentElement.classList.add('displayComment');
+  const commentBtn = document.querySelector('.commentBtn');
+  commentBtn.addEventListener('click', () => {
+    const username = document.querySelector('#commentator').value;
+    const comment = document.querySelector('#comment').value;
+
+    const mealId = meal.idMeal;
+
+    const newComment = {
+      item_id: mealId,
+      username,
+      comment,
+    };
+    addComment(newComment);
```

Comment:
Consider validating the input so that there are no empty users/comments

Reply:
Required not taking effect

Reply:
I just pushed a small implementation for this. Please review and let me know if you need any changes.

Reply:
Go ahead to merge